code/index.html

```diff
@@ -2,21 +2,113 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Krukväxter</title>
     <link rel="stylesheet" href="./style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
-    <header>
+    <header class="header">
       <!-- Some kind of logo and navbar -->
+      <img class="logo" src="./potplantlogo.png" alt="logo"/>
+      <h1 class="titel">Krukväxter</h1>
+      <nav class="navigation-hamburger-container">
+        <div class="hamburger">
+          <div></div>
+          <div></div>
+          <div></div>
+      </nav>    
+      <nav class="navigation-link-container">
+        <a class="links href="">Skötsel</a>
+        <a class="links href="">Blomsteraffärer</a>
+        <a class="links href="">Om</a>
+        <a class="links href="">Kontakt</a>
+      </nav> 
     </header>
+    
+    
+    <section class="main-plant-container">
+      <div class="main-plant-artikle">
```

Comment:
seems like you have a section with the classname _main plant container_ but then just a div with the class name _main plant artikle_ in that. You could just keep one of them, preferable the section as it is a more semnatic element than a div.